### PR TITLE
Fixed Bugs, Added Gemini Ai

### PR DESCRIPTION
**Pull request by iayushanand**
## Checklist

- [x] If code changes were made then they have been tested.
- [x] This PR fixes an issue.

### CODE REVIEW

**Comment by iayushanand** [comment]
I forgot to change back the prefix and remove my id from owners, I will add a commit in a bit fixing these issues.

File: cogs/fun.py
Comment (andrewthederp) [inline cogs/fun.py:1]: look into `commands.Author`

Comment (iayushanand) [reply cogs/fun.py:1]: what do you mean by that?

**Review by iayushanand — COMMENTED** [review]
I said I was gonna remove myself from owner_ids but I think I might need jsk for some stuff while developing the bot

**Review by andrewthederp — APPROVED** [review]
I approve

**Comment by WhoIsConch** [comment]
i forget how to approve pull requests

**Review by WhoIsConch — APPROVED** [review]
i figured it out

**Pull request merged**